handbook/growth.md

```diff
@@ -19,9 +19,9 @@ Posting to social media should follow a [personable tone](https://fleetdm.com/ha
 
 ### Guidelines:
 
-In keeping with our tone, only use hashtags in line and only when it feels natural. If it feels forced, don’t include any.
+In keeping with our tone, use hashtags in line and only when it feels natural. If it feels forced, don’t include any.
 
-Self-promotional tweets are non-ideal tweets.  (Same goes for, to varying degrees, Reddit, HN, Quora, StackOverflow, LinkedIn, Slack, and almost anywhere else.)  See also https://www.audible.com/pd/The-Impact-Equation-Audiobook/B00AR1VFBU
+Self-promotional tweets are not ideal(Same goes for, to varying degrees, Reddit, HN, Quora, StackOverflow, LinkedIn, Slack, and almost anywhere else).  Also see: https://www.audible.com/pd/The-Impact-Equation-Audiobook/B00AR1VFBU
```

Comment:
@timmy-k, can you make this link a related word or phrase for accessibility features to find?
"Also see: https://www.audible.com/pd/The-Impact-Equation-Audiobook/B00AR1VFBU"